docs/lb_radio.rst

```diff
@@ -86,76 +86,87 @@ Syntax Notes
 ------------
 
 The syntax attempts to be intuitive and simple, but it does have some limitations. The artist: entity has the most tricky restrictions
-because it should accept the full name of an artist, so it must be wrapped in ():
+because it should accept the full name of an artist. For latin character sets, the short form for an artist name can be used:
+
+::
+
+  artist:(Blümchen)
+  artist:(The Knife)
```

Comment:
This example should be using the short form I guess?

Reply:
Fixed.